charts/gadget/templates/daemonset.yaml

```diff
@@ -279,10 +279,10 @@ spec:
                 path: config.json
             secretName: gadget-pull-secret
         {{- end }}
-        - configMap:
+        - name: config
+          configMap:
             name: {{ include "gadget.fullname" . }}
-          defaultMode: 400
-          items:
-            - key: config.yaml
-              path: config.yaml
-          name: config
+            defaultMode: 400
+            items:
+              - key: config.yaml
+                path: config.yaml
```

Comment:
Having a quick look at the configmap in git, it seems it only has this file. The idea of the item fields is to only mount in the pod some files/keys, and you use the items to specify which ones. But if you want all, no need to add this.